modules/ROOT/pages/amazon/amazon-sqs-connector-config-topics.adoc

```diff
@@ -0,0 +1,25 @@
+= Amazon SQS Additional Configuration Information - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+
+== Creating the FIFO Queue
+
+To create the FIFO queue, use the *Create queue* operation in the connector, and set the *FifoQueue* and *ContentBasedDeduplication* field values to true.
+
+End the *queue name* with the `.fifo` extension. For example,  `MyTestFIFOQueue.fifo`.
+
+While creating the FIFO queues, choose either of the two regions: US East (Ohio) or US West (Oregon). Amazon only supports these two regions for FIFO queues.
```

Comment:
```suggestion
Choose one of the two supported regions: US East (Ohio), or US West (Oregon).
```

Reply:
Reworded but left in Amazon to since this is an Amazon restriction.